Makefile

```diff
@@ -359,5 +360,18 @@ clean-gui:
 	- rm -rf jujugui/static/gui/build
 
 .PHONY: clean-all
-clean-all: clean-venv clean-pyc clean-gui clean-dist
+clean-all: clean-venv clean-pyc clean-gui clean-dist clean-uitest
 	- rm -rf *.egg-info
+
+.PHONY: clean-uitest
+clean-uitest:
+	- rm -rf uitest
+
+.PHONY: uitest
+uitest: sudo dist
+	@scripts/uitest.sh $(ARGS)
```

Comment:
If we're going to comment out the actual command I'd prefer a user friendly echo.